CODEOWNERS

```diff
@@ -132,6 +134,7 @@
 /gallery/communicationservices-insights/** @microsoft/acs-data
 /gallery/workbook/Microsoft.Communication-CommunicationServices.json @microsoft/acs-data
 /gallery/laws-insights/** @microsoft/loganalytics
+/gallery/update-center-insights/** @microsoft/azure-update-center
```

Comment:
can we change this name to updatecenter-insights? to be consistent with other insights? (will require folder rename for gallery also)

Reply:
done.